### PR TITLE
MMB: Add visa ok information

### PR DESCRIPTION
**Pull request by tbergq**

### CODE REVIEW

File: app/MMB/src/components/Alert.js [hunk 1]
```diff
@@ -33,17 +33,19 @@ const RenderAlert = (props: RenderAlertProps) => (
...
   </View>
 );
 
+const ICON_SIZE = 17;
+
 export default function Alert(props: Props) {
   switch (props.type) {
     case 'danger':
       return (
         <RenderAlert
```
Comment (mrtnzlml) [inline app/MMB/src/components/Alert.js]: Is it time to move `RenderAlert` somewhere else already or not yet? 😊

Comment (tbergq) [reply app/MMB/src/components/Alert.js]: `@kiwicom/mobile-alert`?

Comment (mrtnzlml) [reply app/MMB/src/components/Alert.js]: No, new file! 🙈

Comment (tbergq) [reply app/MMB/src/components/Alert.js]: done

**Pull request merged**